source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Removed the dark mode toggle option for Windows.
+
+MacOS
+^^^^^^^^^^^^^
+-
+
+All Platforms
+^^^^^^^^^^^^^
+- Added Swedish language to the spellchecker.
+- Added a setting to specify the default desktop app download location.
+- Updated the desktop app loading indicator.
+- Restored deeplinking.
+- Switched the dictionary to the one Electron uses to have the ability to fix spelling.
+- Added improvements to be consistent on the use of URL and URL libraries.
+- Removed the Finder functionality from the Desktop App.
```

Comment:
We should never have "removed" in the improvements section. Propose:

Finder functionality has been replaced with in-channel search when using the Mattermost server v5.35 and later.